vagrant.rb

```diff
@@ -202,23 +173,37 @@ def platform(config, pf_id, pf_name)
   machines = machines.sort_by { |k| prio[data[k]['rudder-setup']] + k } # sort by type then by name
   hosts = {}
   host_id=0
-  machines.each do |name|
-    machine = default.merge(data[name])
-    if machine['provider'] == "aws" then
-      aws_machine(config, machines, pf_id, pf_name, name, host_id, machine)
-    else
-      vagrant_machine(config, machines, pf_id, pf_name, name, host_id, machine)
+  machines.each do |host_name|
+    machine = default.merge(data[host_name])
+
+    # Machine name
+    name = pf_name + "_" + host_name
+
+    # Network information
+    network, ip, port = network_info(machine, pf_id, host_id)
+  
+    # Configure
+    config.vm.define name do |cfg|
+      # the provisioning script is generated
+      cfg.vm.synced_folder ".", "/vagrant", disabled: true # disable default sync
```

Comment:
This will remove automatic shared folder for virtualbox for instance. I am pretty sure it should be up by defaut, add a flag in the platform setup to disable it when mandatory (aws, old os)